src/kiln_plugin.h

```diff
@@ -0,0 +1,255 @@
+// --- 7. requestValidatorsExit(validator_pubkeys)
+//
+// V2 selectors
+// --- 8. stake()
+// --- 9. requestExit(shares_count)
+// --- 10. multiClaim(exit_queues, ticket_ids, cask_ids)
+// --- 11. claim(uint256[],uint32[],uint16)
+//
+// LR selectors
+// --- 12. depositIntoStrategy(address,address,uint256)
+// --- 13. queueWithdrawals((address[],uint256[],address)[])
+// --- 14.
+// completeQueuedWithdrawal((address,address,address,uint256,uint32,address[],uint256[]),address[],uint256,bool)
+// --- 15. delegateTo(address,(bytes,uint256),bytes32)
+// --- 16. undelegate(address)
+//
+#define NUM_SELECTORS 17
+extern const uint32_t KILN_SELECTORS[NUM_SELECTORS];
+
+// Selectors available (see mapping above).
+typedef enum {
+    KILN_V1_DEPOSIT = 0,
+    KILN_V1_WITHDRAW,
+    KILN_V1_WITHDRAW_EL,
+    KILN_V1_WITHDRAW_CL,
+    KILN_V1_BATCH_WITHDRAW,
+    KILN_V1_BATCH_WITHDRAW_EL,
+    KILN_V1_BATCH_WITHDRAW_CL,
+    KILN_V1_REQUEST_EXIT,
+    KILN_V2_STAKE,
+    KILN_V2_REQUEST_EXIT,
+    KILN_V2_MULTICLAIM,
+    KILN_V2_CLAIM,
+    KILN_LR_DEPOSIT_INTO_STRATEGY,
+    KILN_LR_QUEUE_WITHDRAWALS,
+    KILN_LR_COMPLETE_QUEUED_WITHDRAWALS,
+    KILN_LR_DELEGATE_TO,
+    KILN_LR_UNDELEGATE,
+} selector_t;
+
+// ****************************************************************************
+// * EIGENLAYER
+// ****************************************************************************
+
+// globals
+
+#define LR_STRATEGIES_COUNT                 11
+#define UNKNOW_LR_STRATEGY                  255
+#define MAX_DISPLAYABLE_LR_STRATEGIES_COUNT (LR_STRATEGIES_COUNT * 3)
+#define ERC20_DECIMALS                      18
+
+extern const char lr_strategy_addresses[LR_STRATEGIES_COUNT][ADDRESS_STR_LEN];
+extern const char lr_erc20_addresses[LR_STRATEGIES_COUNT][ADDRESS_STR_LEN];
+extern const char lr_tickers[LR_STRATEGIES_COUNT][MAX_TICKER_LEN];
+extern const char lr_kiln_operator_address[ADDRESS_STR_LEN];
+
+// ****************************************************************************
+
+// Parameters and state machines for EigenLayer parsing
+
+typedef enum {
+    LR_DEPOSIT_INTO_STRATEGY_STRATEGY = 0,
+    LR_DEPOSIT_INTO_STRATEGY_TOKEN,
+    LR_DEPOSIT_INTO_STRATEGY_AMOUNT,
+    LR_DEPOSIT_INTO_STRATEGY_UNEXPECTED_PARAMETER,
+} lr_deposit_into_strategy_parameters;
+
+typedef enum {
+    LR_QUEUE_WITHDRAWALS_QWITHDRAWALS_OFFSET = 0,
+    LR_QUEUE_WITHDRAWALS_QWITHDRAWALS_LENGTH,
+    LR_QUEUE_WITHDRAWALS__QWITHDRAWALS_STRUCT_OFFSET,
+    LR_QUEUE_WITHDRAWALS__QWITHDRAWALS_STRATEGIES_OFFSET,
+    LR_QUEUE_WITHDRAWALS__QWITHDRAWALS_SHARES_OFFSET,
+    LR_QUEUE_WITHDRAWALS__QWITHDRAWALS_WITHDRAWER,
+    LR_QUEUE_WITHDRAWALS__QWITHDRAWALS_STRATEGIES_LENGTH,
+    LR_QUEUE_WITHDRAWALS__QWITHDRAWALS__STRATEGIES_ITEM,
+    LR_QUEUE_WITHDRAWALS__QWITHDRAWALS_SHARES_LENGTH,
+    LR_QUEUE_WITHDRAWALS__QWITHDRAWALS__SHARES_ITEM,
+    LR_QUEUE_WITHDRAWALS_UNEXPECTED_PARAMETER
+} lr_queue_withdrawals_parameters;
+
+typedef enum {
+    LRCQW_WITHDRAWALS_OFFSET = 0,
```

Comment:
```suggestion
LR_CQW_WITHDRAWALS_OFFSET
```
to be compliant with the rest wdyt ?